sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientBuilder.java

```diff
@@ -234,7 +256,8 @@ public CryptographyClientBuilder httpClient(HttpClient client) {
      * Sets the HTTP pipeline to use for the service client.
      *
      * If {@code pipeline} is set, all other settings are ignored, aside from
-     * ({@link CryptographyClientBuilder#keyIdentifier(String) jsonWebKey identifier} or {@link CryptographyClientBuilder#jsonWebKey(JsonWebKey) json web jsonWebKey}
+     * ({@link CryptographyClientBuilder#keyIdentifier(String) jsonWebKey identifier} or {@link
+     * CryptographyClientBuilder#jsonWebKey(JsonWebKey) json web jsonWebKey}
```

Comment:
Weird line break here. I believe some of the words can fit on the previous line.